docs/schemas/DDECal.yml

```diff
@@ -141,3 +141,17 @@ inputs:
   applycal&#46;*:
     type: 
     doc: ApplyCal sub-step, same as in Predict step. One can pass an h5parm with as many directions as set in "directions" and each direction model is corrupted accordingly `.`
+  phasereference:
+    default: true
+    type: bool
+    doc: Use phase referencing. This setting is only effective when mode=tec or mode=tecandphase `.`
```

Comment:
Could you explain the need for the backtick here? I think it's prettier when `mode=tec` is rendered as code.

Reply:
The tools we use for generating the html pages require the `.` at the end. I've prettyfied the mode=... texts.